assets/components/forms/formHOCs/withError.jsx

```diff
@@ -4,11 +4,13 @@
 
 import React from 'react';
 
+import { type Option } from 'helpers/types/option';
+
 
 // ----- Types ----- //
 
 type AugmentedProps<Props> = Props & {
```

Comment:
What do you think about calling this type `ValidatedProps`?

Reply:
Universally across all the `formHOCs`?

Reply:
If validation is the only 'augmentation' I think it would make more sense yes

Reply:
`asControlled`, `canShow` (coming soon), `withError` and `withLabel` all have this concept of `AugmentedProps`. The name could be updated to make it clearer, but I'd prefer it if it were one that worked across the different HOCs to keep them consistent.